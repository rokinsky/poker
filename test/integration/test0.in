
texas-holdem
omaha-holdem
five-card-draw
texas-holdem 5c6dAcAsQs
omaha-holdem 5c6dAcAsQs
texas-holdem 5c6dAcAsQs Ks4cKd
omaha-holdem 5c6dAcAsQs TsQh9hQc8d
five-card-draw 5s3s4c2h9d8d
texas-holdem 5c6dAcAsQs Ks
omaha-holdem 5c6dAcAsQs TsQh9h
five-card-draw 5s3s4c2h
texas-holdem   2h5c8sAsKc Qs9h
texas-holdem 2h5c8sAsKc   Qs9h
texas-holdem 2h5c8sAsKc Qs2h
omaha-holdem 3d3s4d6hJc Js2dKd2d
five-card-draw 5s3s4c5s9h
