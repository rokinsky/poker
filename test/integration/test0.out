Error: Invalid Input
Error: Invalid Input
Error: Invalid Input
Error: Invalid Input
Error: Invalid Input
Error: Invalid Input
Error: Invalid Input
Error: Invalid Input
Error: Invalid Input
Error: Invalid Input
Error: Invalid Input
Error: Invalid Input
Error: Invalid Input
Error: Invalid Input
Error: Invalid Data (Cards contain duplicates)
Error: Invalid Data (Cards contain duplicates)
Error: Invalid Data (Cards contain duplicates)
